app/src/main/java/nil/nadph/qnotified/hook/InputButtonHook.java

```diff
@@ -273,7 +263,7 @@ public GetMenuItemCallBack() {
         @Override
         protected void afterHookedMethod(MethodHookParam param) throws Throwable {
             if (LicenseStatus.sDisableCommonHooks) return;
-            if (!LicenseStatus.isAsserted() || !CardMsgHook.get().isEnabled()) return;
+            if (LicenseStatus.hasBlackFlags() || !CopyCardMsg.INSTANCE.isEnabled()) return;
```

Comment:
`CardMsgHook.get().isEnabled() || CopyCardMsg.INSTANCE.isEnabled()` please, 开任何一个就开启复制卡片
```suggestion
            if (LicenseStatus.hasBlackFlags() || !(CardMsgHook.get().isEnabled() || CopyCardMsg.INSTANCE.isEnabled())) return;
```

Reply:
这应该是两个功能啊，复制卡片和发送卡片在功能上没有任何重叠和冲突，为啥要二选一，反而会让只想发卡片不想复制卡片的用户迷惑，而且万一随着版本更新这个地方出了问题也不好排查